Lib/test/test_tokenize.py

```diff
@@ -1908,9 +1908,6 @@ def test_random_files(self):
 
         testfiles.remove(os.path.join(tempdir, "test_unicode_identifiers.py"))
```

Comment:
On the other hand, it looks like we _can_ remove this line (and the five lines immediately above it). Now that the tokenizer no longer uses regular expressions, it seems it's able to tokenize this file fine